src/rules/declaration-block-no-ignored-properties/README.md

```diff
@@ -0,0 +1,53 @@
+# declaration-block-no-ignored-properties
+
+Disallow properties has no effect.
+
+```css
+a { display: inline; width: 100px; }
+/**                        ↑
```

Comment:
Minor nitpick, but can you align the arrow with the "w" of width, and then the "y" of property with the arrow. We've so many docs that we have strict conventions so that they are easier for users to scan.